modules/serverless-connect-sink-source-odc.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+//  * serverless/channels/connect-channels-event-source.adoc
+// * serverless/event_sources/srvls-connecting-source-sinks.adoc
+
+[id="serverless-connect-sink-source-odc_{context}"]
+= Connect an event source to a sink using the Developer perspective
+
+You can create multiple event source types in {product-title} that can be connected to sinks.
+
+A sink is an addressable resource that can receive incoming events from other Knative Eventing resources. Knative services, channels, and brokers are all examples of sinks.
+// Add functions as services to the list for tech preview
+
+.Prerequisites
+
+To connect an event source to a sink using the *Developer* perspective, ensure that:
+
+* The {ServerlessOperatorName}, Knative Serving, and Knative Eventing are installed on your {product-title} cluster.
+* You have logged in to the web console.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have created a sink.
+
+.Procedure
+
+. Create an event source of any type.
+. In the *Developer* perspective, navigate to *Event Sources*.
+. In the *Sink* section of the *Event Sources* form view, select *Resource*. Then use the drop-down list to select your sink.
```

Comment:
I was not able to follow these steps. The page, for me, after going into Event Sources, appears the event sources in boxes

Reply:
Updated the steps to make it clearer that you need to create an event source and then choose the sink